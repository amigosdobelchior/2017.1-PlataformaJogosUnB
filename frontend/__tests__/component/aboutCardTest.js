import React from 'react';
import {Card, Grid, Image} from "semantic-ui-react";
import renderer from 'react-test-renderer';


test('Test render AboutCard', () => {
  const component = renderer.create(
      <Card>
          <Card.Content>
              <Card.Header>
                  {"Titulo"}
              </Card.Header>
          </Card.Content>
          <Card.Content>
              <Card.Description>
                  <Grid centered columns={5} divided>
                      <Grid.Row>
                          <Grid.Column>
                              {"description"}
                          </Grid.Column>
                          <Grid.Column>
                              <Image centered size="medium" src="./image1.jpg" />
                          </Grid.Column>
                      </Grid.Row>
                  </Grid>
              </Card.Description>
          </Card.Content>
      </Card>
     );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
