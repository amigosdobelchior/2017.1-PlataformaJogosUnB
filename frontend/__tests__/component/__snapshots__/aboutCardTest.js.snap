// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render AboutCard 1`] = `
<div
  className="ui card"
  href={undefined}
  onClick={[Function]}
>
  <div
    className="content"
  >
    <div
      className="header"
    >
      Titulo
    </div>
  </div>
  <div
    className="content"
  >
    <div
      className="description"
    >
      <div
        className="ui centered divided five column grid"
      >
        <div
          className="row"
        >
          <div
            className="column"
          >
            description
          </div>
          <div
            className="column"
          >
            <img
              alt={undefined}
              className="ui medium centered image"
              height={undefined}
              src="./image1.jpg"
              width={undefined}
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
